Resources/Prototypes/Species/ipc.yml

```
@@ -57,42 +59,42 @@
       points: 1
       required: false
     Chest:
-      points: 1
+      points: 6
       required: true
       defaultMarkings: [ MobIPCTorsoDefault ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)